doc/releases/release-notes-3.5.rst

```diff
@@ -2060,6 +2060,9 @@ Libraries / Subsystems
     supported in function handlers by setting ``mg_translate_error`` of
     :c:struct:`mgmt_group` when registering a group. See
     :c:type:`smp_translate_error_fn` for function details.
+* shell
+
+  * Added an application shell module with helpers common to all applications.
```

Comment:
you do not want to be changing release notes of an older release...

Reply:
Good catch. I was just trying to clean up things from last year and got a notification that the other pr would be closed.

I think `mallinfo()` is fairly useful, so I wanted to try and rescue it, if possible. It's not ISO C, but... still useful.

The problem is, it's not working with picolibc. Have not looked into it into too much detail.

Minimal Libc is fine though.